src/types.ts

```diff
@@ -60,6 +60,16 @@ export interface QuizMetadata {
   topic?: string;
 }
 
+export interface QuestionBucket {
+  bucketStartIndex: number,
+  bucketEndIndex: number,
+  hasBeenFetched: boolean
```

Comment:
?
```suggestion
  fetched: boolean
```

Reply:
I'm keeping it `isFetched` like we used to do in plio's code for boolean naming